README.md

```diff
@@ -65,23 +85,30 @@ node addFields.js
 # Development
 
 ## Contributions, bug reports and suggestions
+
 If you would like to contribute or send feedback, please [create an issue](https://github.com/Fysikteknologsektionen/chalmers-course-stats/issues/new) first.
 
 ## Set up development environment
+
 Run these commands:
+
 ```bash
 cd /path/to/development/directory
 git clone https://github.com/Fysikteknologsektionen/chalmers-course-stats/
 npm install
 ```
+
 To start the server run (in separate terminals)
-- `mongod`
+- `mongod` (use `mongod --dbpath` to run the database from the same folder)
```

Comment:
Could you add context of when you need to run with this flag?

Reply:
Added some context to the readme. It's when you dont want it to run on the default path (which on windows seems to be C:/data/db)